src/index.test.js

```diff
@@ -112,6 +112,55 @@ describe("#index - src", () => {
     expect(Reports["elastic-search"].mock.calls).toHaveLength(0);
   });
 
+  test("Success calls with custom exporter", async () => {
+    jest.mock("moment", () => () => ({
+      format: () => "2020–01–30T12:34:56+00:00"
+    }));
+
+    const Format = require("./format");
+    jest.mock("./format");
+    Format.mockReturnValue({result: true});
+
+    const Index = require("./index");
```

Comment:
```suggestion
    const RQAInstance = require("./index");
```

or `Instance` or `RestQAInstance`.